tests/test_browser.py

```diff
@@ -1805,7 +1805,7 @@ def test_clientside_vertex_arrays_es3(self):
     self.btest('clientside_vertex_arrays_es3.c', reference='gl_triangle.png', args=['-s', 'FULL_ES3=1', '-s', 'USE_GLFW=3', '-lglfw', '-lGLESv2'])
 
   def test_emscripten_api(self):
-    self.btest('emscripten_api_browser.cpp', '1', args=['-s', 'EXPORTED_FUNCTIONS=_main,_third', '-lSDL'])
```

Comment:
Sorry I may not have the sufficient context yet.. In the existing code, some calls to `btest` have the expected value of `1`, while other have `0` or something else, but the new `btest_exit` doesn't contain any. Does deleting these different expected values not matter then? Also, some existing test cases report different values depending on tests using `REPORT_RESULT`, but they are all deleted. These values don't matter and can be deleted safely?

Reply:
`btest_exit` can detect non-zero exit codes but the default and simplest thing to do is to expect an exit code of zero.   What I've done in this change is modified some of the tests to report zero on success and non-zero on failure, at least where it makes sense.  As far as I can tell, and this is where you can help me out, I don't think I've modified the semantics of any of the tests.   They still report success and failures in the same cases. 

For an example of a test where I continue to use the non-zero expectation see `test_emscripten_api_infloop` which as assert_returncode=7.   I can't remember why I didn't change that one in particular, but it shows how btest exit works and how it can be used to assert on non-zero exit codes.